operator_ui/src/components/Link.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import { Link as ReactStaticLink } from 'react-router-dom'
+import {
+  createStyles,
+  Theme,
+  withStyles,
+  WithStyles
+} from '@material-ui/core/styles'
+import Typography from '@material-ui/core/Typography'
+import { grey } from '@material-ui/core/colors'
+import { ThemeStyle } from '@material-ui/core/styles/createTypography'
+import { PropTypes } from '@material-ui/core'
+import classNames from 'classnames'
+
+type Variant = ThemeStyle | 'srOnly'
```

Comment:
Any ideas on a better way to do this? It's a union type that isn't exported from the lib

https://github.com/mui-org/material-ui/blob/89687f38cae750650555772ba4d821c9084d8dfc/packages/material-ui/src/Typography/Typography.d.ts#L5

Reply:
No, but I think this is a great approach.